src/Types.jl

```diff
@@ -928,6 +936,27 @@ function ensure_resolved(manifest::Manifest,
     pkgerror(msg)
 end
 
+# copied from REPL to efficiently expose if any score is >0
+function fuzzysort(search::String, candidates::Vector{String})
+    scores = map(cand -> (REPL.fuzzyscore(search, cand), -Float64(REPL.levenshtein(search, cand))), candidates)
+    candidates[sortperm(scores)] |> reverse, any(s -> s[1] > 0, scores)
+end
```

Comment:
from Slack- it would be better to use REPL's `levsort` instead (https://github.com/JuliaLang/julia/blob/f3e3554910a79fa38d5662d14abe04e137075e18/stdlib/REPL/src/docview.jl#L652-L660), which is what the REPL uses for corrections: https://github.com/JuliaLang/julia/blob/f3e3554910a79fa38d5662d14abe04e137075e18/stdlib/REPL/src/docview.jl#L700-L707.

This does a lot better on `DifferentielEquations`:
```julia
julia> levsort("DifferentielEquations", all)
3-element Vector{String}:
 "DifferentialEquations"
 "DifferenceEquations"
 "DifferentialEvolutionMCMC"
```
vs
```julia
julia> fuzzysort("DifferentielEquations", all)
(["DifferentialEvolutionMCMC", "DifferentialDynamicsModels", "DifferentiableStateSpaceModels", "DifferentialEquations", "SimpleDifferentialOperators", "DiscreteDifferentialGeometry", "DifferentialForms", "DifferentialDynamicProgramming", "DifferenceEquations", "AbstractDifferentiation"  …  "BenchmarkingEconomicEfficiency", "BayesianNonparametricStatistics", "ApproxFunOrthogonalPolynomials", "GAP_pkg_normalizinterface_jll", "ReinforcementLearningEnvironmentAtari", "MultivariateOrthogonalPolynomials", "BlackBoxOptimizationBenchmarking", "SemiclassicalOrthogonalPolynomials", "OpenTelemetryExporterOtlpProtoGrpc", "ReinforcementLearningEnvironmentClassicControl"], false)
```